micro_ros_setup/config/zephyr/generic/create.sh

```diff
@@ -0,0 +1,52 @@
+# To be removed later, please use package.xml
+sudo apt update
+sudo apt install --no-install-recommends git cmake ninja-build gperf ccache dfu-util device-tree-compiler wget python3-pip python3-setuptools python3-tk python3-wheel xz-utils file make gcc gcc-multilib software-properties-common -y
+
+wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | sudo apt-key add -
+sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
+sudo apt update
+sudo apt install cmake -y
+
+# pip3 install --user -U west
+export PATH=~/.local/bin:"$PATH"
+
+pushd $FW_TARGETDIR >/dev/null
+   
+    west init zephyrproject
+    pushd zephyrproject >/dev/null
+        west update
+    popd >/dev/null
+
+    pip3 install -r zephyrproject/zephyr/scripts/requirements.txt
+
+    export TOOLCHAIN_VERSION=zephyr-toolchain-arm-0.11.2-setup.run
+    wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.11.2/$TOOLCHAIN_VERSION
+    chmod +x $TOOLCHAIN_VERSION
+    ./$TOOLCHAIN_VERSION -- -d $(pwd)/zephyr-sdk -y
+
+    rm -rf $TOOLCHAIN_VERSION
+
+
+    # Temporal until driver in mainstream
+    pushd zephyrproject/zephyr >/dev/null
+        git remote add eprosima https://github.com/eProsima/zephyr
+        git fetch --all
+        git checkout remotes/eprosima/feature/driver_vl53l1
+    popd >/dev/null
+
+    export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
+    export ZEPHYR_SDK_INSTALL_DIR=$FW_TARGETDIR/zephyr-sdk
+
+    # Import repos
+    vcs import --input $PREFIX/config/$RTOS/$PLATFORM/board.repos >/dev/null
+
+    # ignore broken packages
+    touch mcu_ws/ros2/rcl_logging/rcl_logging_log4cxx/COLCON_IGNORE
+    touch mcu_ws/ros2/rcl/COLCON_IGNORE
```

Comment:
Aren't these needed??

Reply:
Well, we are not using logging for C++ because we have not tested C++ nor RCLCPP on Zephyr.

Regarding the ros2/rcl: it is replaced by micro-ROS/rcl where some changes regarding security folders have been done. You can check this changes here: https://github.com/micro-ROS/rcl/commits/dashing

PR to integrate this fork on ros2 repos are here: https://github.com/ros2/rcl/pull/544 and https://github.com/ros2/rcutils/pull/195

Reply:
thanks for the info. btw, looks as if those PR's are unlikely to be merged in their current state.

Reply:
I know... I think that soon or later we should talk about what to do with rcutils for micro-ROS targets without filesystem.

It is something to talk maybe in the next techical meeting @julibert @FranFin